apps/nerves_hub_www/lib/nerves_hub_www_web/views/device_view.ex

```diff
@@ -56,4 +56,12 @@ defmodule NervesHubWWWWeb.DeviceView do
   def tags_to_string(tags), do: tags
 
   defdelegate device_status(device), to: Presence
+  
+  def take_device_tags(device, amount) do
+    Enum.take(device.tags, amount)
+  end
+  
+  def count_device_tags(device) do
+    Enum.count(device.tags)
+  end
```

Comment:
is it better to create methods this simple, or just use `Enum` in templates?

Reply:
I like the methods here for now. If we start to do a lot of really similar duplicate style methods then we can think about consolidation.